src/test/java/uk/nickbdyer/chatServer/ChatRoomTest.java

```diff
@@ -0,0 +1,14 @@
+package uk.nickbdyer.chatServer;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class ChatRoomTest {
+    
+    @Test
+    public void containsKnowSentencesWhenInitialized() {
```

Comment:
_Super picky_ now, but keep in mind that this test description is misleading when you look at the actual code.
The description mentions that the chat room should contain sentences, but it actually just verifies the amount of messages.
Additionally it only verifies the absence of sentences.